### ISSUE TITLE
Feature request: to have more featurebox category with the same template

### ISSUE THREAD

**Issue by Jimmi08** [e107inc/e107#1797]
Now you can have just one category with for example tabs template. 
But I have plugin for projects, where each project can have its own tabs (or other featurebox template). For each project I want to select featurebox category but I can't because I can't create so many featurebox categories as projects are with the same template. 

Why there is this restriction? Now category = template or I am missing something.

Thanks

**Comment by Jimmi08** [comment]
I know about this. But I don't know numbers of needed categories and with each new category to change the code is a bit non practical. I was able to find workaround for tabs, I am just curious if this is purpose or mistake or not finished.

**Comment by rica-carv** [comment]
Right now is the only way to go.
There's no limit to categories, if you name them different from the existing ones.

If this is a unfinished code or mistake, i simply don't know...

**Comment by Jimmi08** [comment]
No, there is limit to categories. You can have only so much categories as you have created templates in code.  Otherwise you get SQL error, because this:
![image](https://cloud.githubusercontent.com/assets/5429548/17520466/1eaa182e-5e50-11e6-9e0f-b2c84ec53042.png)

**Comment by rica-carv** [comment]
@Jimmi08 Right, that's why i've stated before that you have to create a category template for each new category, even if that template already exists with other category.
Each template has is own category:
Category 1 = Template 1
Category 2 = Template 2
and so on....

Of course you could have:
Category 1 =  Template 1
Category 2 = Template 2 wich is = Template 1

But each unique category name has to have also its unique template name.
And that's infinite....

**Comment by rica-carv** [comment]
@Jimmi08 I know we are going in circles, since is quite odd to have a unique category linked to a unique template.
It's a bit redundant why have category, or why have templates....
As i see it, a category should have many templates possibilities, or should it be the othe way around?

**Comment by Moc** [comment]
Is this issue still relevant?

**Comment by CaMer0n** [comment]
@Moc Featurebox was mostly rewritten by Miro. I'm guessing it was intentional due to conflicts of some kind.

**Comment by Jimmi08** [comment]
@CaMer0n @Moc 
If there are no plans to change this, this can be closed. I don't need it anymore.
